src/main/frontend/date.cljs

```diff
@@ -77,14 +77,6 @@
     (catch js/Error _e
       nil)))
 
-(defn ISO-string
```

Comment:
These were all unused but if anything is helpful for development, I can add them to ignore